apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx

```diff
@@ -48,12 +48,15 @@ const PageInformation = () => {
     };
     useEffect(() => {
         fetchPageInfo();
-        fetchPageHistory();
+        fetchPageHistory().then();
```

Comment:
Issue: `.then()` doesn't do anything

Reply:
That is just to satisfy the lint warning